core/templates/pages/profile-page/profile-page.controller.ts

```diff
@@ -55,8 +55,10 @@ angular.module('oppia').directive('profilePage', [
           };
           ctrl.$onInit = function() {
             LoaderService.showLoadingScreen('Loading');
-            $http.get(profileDataUrl).then(function(response) {
-              var data = response.data;
+
+            let fetchProfile = () =>
```

Comment:
Perhaps better to call it fetchProfileData.

Reply:
Done.